client/src/pages/contentPages/Groups.js

```diff
@@ -1,23 +1,278 @@
-import React from 'react';
-import { Grid, Typography, Container } from '@material-ui/core';
+import React, { useState, useEffect, useMemo, useCallback } from 'react';
+import { Grid, Typography, Container, Button, Modal, FormHelperText, MenuItem, Select, TextField } from '@material-ui/core';
+import {
+  baseStyle,
+  activeStyle,
+  acceptStyle,
+  rejectStyle,
+} from '../../components/Group/GroupPicture.styles';
+import { Tooltip, Box} from '@material-ui/core';
+import { useDropzone } from 'react-dropzone';
+import axios from 'axios';
+
 import { makeStyles } from '@material-ui/core/styles';
 import GroupCard from '../../components/Group/GroupCard';
-import axios from 'axios';
+import defaultImage from '../../images/upload_placeholder.png';
+import { useSnackbar } from 'notistack';
 import { useGlobalContext } from '../../context/studyappContext';
+import * as actions from '../../context/actions';
 
 const useStyles = makeStyles((theme) => ({
     cardGrid: {
       paddingTop: theme.spacing(8),
       paddingBottom: theme.spacing(8),
     },
+    button: {
+      color: '#FFF',
+      background: theme.palette.primary.gradient,
+      marginTop: "60px",
+      marginRight: "60px",
+      textTransform: 'none',
+      height: '3rem',
+  },
+    paper: {
+      position: 'fixed',
+      width: 475,
+      backgroundColor: theme.palette.background.paper,
+      boxShadow: theme.shadows[5],
+      padding: theme.spacing(2, 4, 3),
+      borderRadius: 4,
+      outline: 'none'
+
+    },
+
+    groupImageContainer: {
+      flex: 1,
+      display: 'flex',
+      flexDirection: 'column',
+      justifyContent: 'center',
+      alignItems: 'center',
+      paddingTop: "10px",
+      paddingBottom: "10px"
+    },
+    large: {
+      height: "300px",
+      width: "400px",
+    },
+    uploading: {
+      opacity: 0.5,
+      height: "300px",
+      width: "400px"
+    },
+    
   }));
 
+  function getModalStyle() {
+    const top = 50;
+    const left = 50;
+  
+    return {
+      top: `${top}%`,
+      left: `${left}%`,
+      transform: `translate(-${top}%, -${left}%)`,
+    };
+  }
+
+
+
 const Groups = () => {
     const classes = useStyles();
-    const {userGroups, dispatch} = useGlobalContext();
+    const {userGroups, userCourse, dispatch} = useGlobalContext();
+
     const {groups, courseGroups} = userGroups;
     const [suggestedGroups, setSuggestedGroups] = React.useState([]);
```

Comment:
You can change this to just be `useState`, the function is being imported using destructuring on line 1, along with the React import.